### PR TITLE
HOMEWORK4 , Done

### PR DESCRIPTION
**Pull request by AfafSaleh55**

### CODE REVIEW

**Review by SalimNoor — COMMENTED** [review]
Try to improve the same code using better options we discussed like using loops / in-built functions / dictionaries

File: Homework4/HomeWork4:1.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+var Cities = "Mahkh"
```
Comment (azfay3) [inline Homework4/HomeWork4:1.playground/Contents.swift:3]: No spaces!

File: Homework4/HomeWork4:1.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+var citiy = [String]()
+
+citiy.append(" Jeedh ")
+citiy.append(" Mahkh ")
```
Comment (azfay3) [inline Homework4/HomeWork4:1.playground/Contents.swift]: It is different than this!

**Review by azfay3 — CHANGES_REQUESTED** [review]
Remove spaces

**Comment by azfay3** [comment]
Great job!

**Pull request closed without merging**